source/deployment/sunstone_setup/suns_advance.rst

```diff
@@ -138,7 +140,10 @@ Next thing we have to do is configure the virtual host that will run our Sunston
       </Directory>
     </VirtualHost>
 
-.. note:: When you're experiencing login problems you might want to set ``PassengerMaxInstancesPerApp 1`` in your passenger configuration or try memcached since Sunstone does not support sessions across multiple server instances.
+.. note::
+    When you're experiencing login problems you might want to set ``PassengerMaxInstancesPerApp 1`` in your passenger configuration or try memcached since Sunstone does not support sessions across multiple server instances.
+
+    The <username> of the virtualhost must be the user who installed sunstone and the owner of /var/lib/one/.one/sunstone_auth
```

Comment:
For instance, here I'd add that for default OpenNebula installations, <username> is oneadmin.